rpmvenv/extensions/loader.py

```diff
@@ -21,10 +21,11 @@ class InvalidDependency(Exception):
 
 def load_extensions(whitelist=()):
     """Get an iterable of extensions in order."""
-    whitelist = tuple(set(('core',) + tuple(whitelist)))
+    whitelist = tuple(('core',) + tuple(whitelist))
+    unique_whitelist = tuple(set(whitelist))
```

Comment:
I don't believe this actually changes any behaviour. You have decomposed the one liner into two lines but the overall logic is identical. Calling `tuple(set(tuple()))` is the same as calling `tuple(set())` when the inner target is already a tuple. An intermittent ordering issue would not be resolved with this change.